Sources/Testing/Running/Configuration.TestFilter.swift

```diff
@@ -28,42 +28,52 @@ extension Configuration {
       case excluding
     }
 
-    /// An enumeration describing the various kinds of test filter.
-    fileprivate enum Kind: Sendable {
+    /// An enumeration describing a predicate that a test filter may specify.
+    fileprivate enum Predicate: Sendable, Codable {
```

Comment:
I think `Kind` here is probably still appropriate and is consistent with our other nested-enum types.